### PR TITLE
Bug fixed from commit https://github.com/SubtitleEdit/subtitleedit/pull/495

### PR DESCRIPTION
**Pull request by ivandrofly**
/cc @niksedk Safe to merge

### CODE REVIEW

File: src/Logic/Forms/FixCommonErrorsHelper.cs [hunk 1]
```diff
@@ -451,14 +451,7 @@ internal static string FixShortLines(string text)
...
                 s = s.TrimStart('-');
-                if (!s.Contains('.') &&
-                    !s.Contains('?') &&
-                    !s.Contains('!') &&
-                    !s.Contains(':') &&
-                    !s.Contains(';') &&
-                    !s.Contains('-') &&
-                    !s.Contains('♪') &&
-                    !s.Contains('♫') &&
+                if (s.IndexOfAny(new[] { '.', '?', '!', ':', ';', '-', '♪', '♫' }) < 0 &&
```
Comment (ivandrofly) [inline src/Logic/Forms/FixCommonErrorsHelper.cs]: Faster than ```contains + contains```...

File: src/Logic/Forms/RemoveTextForHI.cs [hunk 2]
```diff
@@ -162,7 +162,7 @@ public string RemoveColon(string text)
                             if (remove)
                             {
                                 var content = line.Substring(indexOfColon + 1).Trim();
-                                if (content.Length > 0)
+                                if (HtmlUtil.RemoveHtmlTags(content, true).Length > 0)
```
Comment (ivandrofly) [inline src/Logic/Forms/RemoveTextForHI.cs]: Check if content is not only tag:
```Narrator: <i></i>```

File: src/Logic/Forms/FixCommonErrorsHelper.cs (hunk 1 above)
Comment (niksedk) [inline src/Logic/Forms/FixCommonErrorsHelper.cs]: :)

**Comment by niksedk** [comment]
a bit too much in same pr...

**Comment by ivandrofly** [comment]
:+1: will repull

**Pull request closed without merging**